apps/zotonic_mod_seo/src/support/seo_breadcrumb.erl

```diff
@@ -23,6 +23,8 @@
     find/2
 ]).
 
+-define(MAX_HASPART, 3).
```

Comment:
This seemed a low number; I wondered how many items I would deem good for a summary representation. I guess I would take 10 but that might be heavy.